### ISSUE TITLE
Custom icons from gallery 

### ISSUE THREAD

**Issue by unn113** [MM2-0/Kvaesitso#336]
Let users pick custom icons from gallery / file explorer app

**Comment by exstntlmsnthrp** [comment]
You can select custom icons for each app, if you wish. So I'm not sure I understand what you're trying to accomplish. 

Can you please explain a little more what you're trying to do and unable to?

**Comment by MM2-0** [comment]
> Can you please explain a little more what you're trying to do and unable to?

You can only pick icons from installed icon packs. They want to be able to pick any image file as custom icon

**Comment by unn113** [comment]
> > Can you please explain a little more what you're trying to do and unable to?
> 
> You can only pick icons from installed icon packs. They want to be able to pick any image file as custom icon

Exactly, i have my own iconpack, but i have my icons in .png, I'd like to pick them from gallery.

**Comment by Crazy-Unicooorn** [comment]
That would be my wish too!

**Comment by MichaelLy03** [comment]
I also wanted the feature baked in but I found another app that does what you guys are looking for. Alembicons on fdroid you can pick an image for whatever app you want and creates a icon pack for you.

**Comment by StioStudio** [comment]
> I also wanted the feature baked in but I found another app that does what you guys are looking for. Alembicons on fdroid you can pick an image for whatever app you want and creates a icon pack for you.

Thanks, that is quite useful!

**Comment by hispka** [comment]
> I also wanted the feature baked in but I found another app that does what you guys are looking for. Alembicons on fdroid you can pick an image for whatever app you want and creates a icon pack for you.

The problem with alembicons is whenever you have a new app which isnt supported by the icon pack, you have to manually create and export a new icon pack so it's kinda a janky workaround, it'd be much nicer if the feature was just baked into the launcher.
